index.html

```diff
@@ -236,4 +236,4 @@ <h3>Контакты</h3>
     ></noscript>
     <!-- /Yandex.Metrika counter -->
   </body>
-</html>
+</html>
```

Comment:
The eol issue is still here and in each of updated files. Please fix them and it seems like the PR should be merged.